### DISCUSSION TITLE
Preview Feedback: C# 12 Default values in lambda expressions

### DISCUSSION THREAD

**Discussion by KathleenDollard** [dotnet/csharplang#7111] [General]
Default values in lambda expressions are included starting in C# 12 in preview 3.

You can use this discussion for feedback as you use the feature.

You can learn more about default values in lambda expressions at [What's new in C# 12](https://learn.microsoft.com/dotnet/csharp/whats-new/csharp-12#default-lambda-parameters).

**Comment by AlgorithmsAreCool** [comment]
For other folks that were confused by this feature such as myself; it seems the motivating use case is for use with aspnet minimal APIs.

**Reply by DavidArno** [reply]
Thanks. I was genuinely confused as to what was the point of this feature. Appreciate you sharing.
